cassandra/cassandra-docker.sh

```diff
@@ -286,7 +286,7 @@ if [ -n "$KEYSTORE_PASSWORD_FILE" ]; then
 fi
 if [ ! -n "$KEYSTORE_FILE" ]; then
     KEYSTORE_FILE="${KEYSTORE_DIR}/.keystore"
-    KEYSTORE_PASSWORD=${KEYSTORE_PASSWORD:-$(head /dev/urandom -c 512 | tr -dc A-Z-a-z-0-9 | head -c 17)}
+    KEYSTORE_PASSWORD=${KEYSTORE_PASSWORD:-$(openssl rand -base64 512 | tr -dc A-Z-a-z-0-9 | head -c 17)}
```

Comment:
You don't need the `tr` and the `head` if you are using `openssl rand`.  I'm not sure what your end result is - are you trying to have a 16 character value?  If so, then you'll need to adjust the numeric argument to `openssl rand` i.e. just try it out from the command line and see what happens, and view the man page for openssl rand.